### PR TITLE
Tweaked the script to prefix all single char variables with `o`

### PR DESCRIPTION
**Pull request by abiosbug**
Hi,

as explained in issue #3 I went ahead and updated your script to work with 2 character variables.  I used the prefix `O` as it seems an uncommon prefix in most complex scripts I've seen, adjusted `:autoscan` to `:scan` to save on space and replaced `:Re` with `:Reset` so it doesn't need a custom name anymore.

I hope you like it! We in our Singularity company sure do ;-)

### CODE REVIEW

**Comment by abiosbug** [comment]
Currently testing, something seems off atm

**Comment by abiosbug** [comment]
Hi,

I've been testing some more, and the windings characters seem really evil.  I mean, even the Github diff viewer crashes on those characters, failing to display the diff properly.  So to be honest, I will never include any of that *cough* into my ships, way too risky given that character encoding issues have been causing headaches for well over 20 years now.

TL;DR; IMHO the visual gimmick is totally NOT worth the risk of bugging out your ship and your game.

**Comment by DerPfandadler** [comment]
I will adjust the device fields and adjust string assets.
